dev-python/pylibmc/pylibmc-1.5.1-r1.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+PYTHON_COMPAT=( python{2_7,3_4,3_5} )
+
+inherit distutils-r1
+
+DESCRIPTION="Libmemcached wrapper written as a Python extension"
+HOMEPAGE="http://sendapatch.se/projects/pylibmc/ https://pypi.python.org/pypi/pylibmc"
+# One image is missing from the doc at PyPI
+# https://github.com/lericson/pylibmc/pull/221
+SRC_URI="https://github.com/lericson/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc test"
+
+RDEPEND=">=dev-libs/libmemcached-0.32"
+# Older sphinx versions fail to compile the doc
+# https://github.com/sphinx-doc/sphinx/issues/3266
+DEPEND="${RDEPEND}
+	doc? ( >=dev-python/sphinx-1.5.1[${PYTHON_USEDEP}] )
+	test? (
+		net-misc/memcached
+		dev-python/nose[${PYTHON_USEDEP}]
+	)"
+
```

Comment:
then add
```
pkg_setup() {
	use doc && DISTUTILS_ALL_SUBPHASE_IMPLS=( 'python*' )
}
```
and finally readd `pypy` (and if you're ultra adventurous, try adding `pypy3`)